features/networking/multus.feature

```diff
@@ -1033,13 +1034,13 @@ Feature: Multus-CNI related scenarios
   Scenario: Multus default route overwrite
     # Make sure that the multus is enabled
     Given the multus is enabled on the cluster
-
+    Given the default interface on nodes is stored in the :default_interface clipboard
     # Create the net-attach-def via cluster admin
     Given I have a project
     When I run oc create as admin over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/NetworkAttachmentDefinitions/ipam-static.yaml" replacing paths:
       | ["metadata"]["namespace"]  | <%= project.name %> | 
       | ["metadata"]["name"]       | bridge-static       |
-      | ["spec"]["config"]| '{ "cniVersion": "0.3.0", "type": "bridge", "master": "ens3", "ipam": {"type":"static","addresses": [{"address": "22.2.2.22/24","gateway": "22.2.2.1"}]}}' |
+      | ["spec"]["config"]| '{ "cniVersion": "0.3.0", "type": "bridge", "master": "<%= cb.default_interface %>", "ipam": {"type":"static","addresses": [{"address": "22.2.2.22/24","gateway": "22.2.2.1"}]}}' |
```

Comment:
I think we can delete the parameter `master` as we talked before since we cannot sure the primary interface of all nodes are same .

Reply:
@zhaozhanqi Fix to delete master parameter, thanks!

Reply:
@weliang1 lgtm, except if you can try it to run locally post master deletion to see if cranking doesn't impact?

Reply:
@anuragthehatter Test passed log: http://pastebin.test.redhat.com/860020, thanks!

Reply:
/LGTM